src/bun.js/api/server/RequestContext.zig

```diff
@@ -1483,10 +1515,45 @@ pub fn NewRequestContext(comptime ssl_enabled: bool, comptime debug_mode: bool,
                 return;
             }
 
+            
             if (response_value.as(JSC.WebCore.Response)) |response| {
                 ctx.response_jsvalue = response_value;
                 ctx.response_jsvalue.ensureStillAlive();
                 ctx.flags.response_protected = false;
+
+                // () => new Response(HTMLBundle)
+                if(response.html_bundle_route) |route_ref| {
```

Comment:
This doesn't handle AbortSignal? We could probably get away with that for an initial release